python_scripts/trees_classification.py

```diff
@@ -55,9 +55,12 @@ def plot_decision_function(fitted_classifier, range_features, ax=None):
         np.arange(*range_features[feature_names[0]], plot_step),
         np.arange(*range_features[feature_names[1]], plot_step),
     )
+    grid = pd.DataFrame(np.c_[xx.ravel(), yy.ravel()],
```

Comment:
I think that in this case, we should update the function to a display: https://github.com/glemaitre/datascience_middle_course/blob/main/02_linear_models/helper/plotting.py#L59

I think that the warning is more on our side here.